lib/raven/integrations/sidekiq.rb

```diff
@@ -22,6 +27,9 @@ def call(ex, context)
         :extra => { :sidekiq => context },
         :culprit => culprit_from_context(context)
       )
+    ensure
```

Comment:
True, however it was there in the code before your change: https://github.com/getsentry/raven-ruby/pull/637/files#diff-f18bf682cd7bdb47e4e6bdd40c800adcL16  and it does seem like it might be good to account for the possibility of the exception reporting code itself to also raise an exception, however unlikely and annoying that might be. There's also no obvious downside to this approach that I can think of.